system/Router/RouteCollection.php

```diff
@@ -313,7 +313,7 @@ public function __construct(FileLocator $locator, Modules $moduleConfig, Routing
      *
      * @return $this
      */
-    public function loadRoutes(string $routesFile = APPPATH . 'Config/Routes.php')
+    public function loadRoutes(string $routesFile = APPPATH . 'Config' . DIRECTORY_SEPARATOR . 'Routes.php')
```

Comment:
Then `$routesFile` always must be normalized (windows style if on Windows).
At least, please add doc comment about it.

And do we need to add a coding rule to always use normalized path in every place?
To be honest, I want to use `/` as a directory separator and want to write like `Config/Routes.php`.

Reply:
> To be honest, I want to use `/` as a directory separator and want to write like `Config/Routes.php`.

Ok. I agree with that. I will change the commit to do this validation